docs/DeveloperGuide.md

```diff
@@ -1,37 +1,1036 @@
+|---------------------------------------------------------------------|----------------|------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [schedule /update `<day number>` `<plan number>`](#update-schedule) | update         | `<day number>` Number representing the day. <br/>`<plan number>` Index of the plan stored in planList. This is the plan to be scheduled for the day. |
+| [schedule /list](#view-schedule)                                    | list           |                                                                                                                                                      |
+| [schedule /clear `<day number>`](#clear-schedule-for-a-day)         | clear          | `<day number>` Number representing the day.                                                                                                          |
+| [schedule /clearall](#clear-schedule-for-the-week)                  | clearall       |                                                                                                                                                      |
+
+The `<day number>` range from 1 to 7. The meaning of each day number is explained in the table below.
+
+| Day Number | Meaning   |
+|------------|-----------|
+| 1          | Monday    |
+| 2          | Tuesday   |
+| 3          | Wednesday |
+| 4          | Thursday  |
+| 5          | Friday    |
+| 6          | Saturday  |
+| 7          | Sunday    |
+
+This method will further evaluate the `<userAction>` and call the constructor of `ScheduleCommand` class by 
+passing relevant parameters related to schedule to the constructor. If the `<userAction>` is null or incorrect, 
+an `InvalidCommandException` will be thrown.
+If the `<parameters>` of certain commands are not specified or met, an `InvalidScheduleException` will be thrown.
+
+---
+#### Update Schedule
+A summary of the general procedure of updating a plan for a particular day to the schedule in WerkIt! is as follows:
+1. User enters the command `schedule /update <day number> <plan number>`.
+2. If there are no plan being scheduled for the day, a new Day object is created and stored in the application.
+   If there is an existing plan scheduled for that particular day, the Day object that had already been created,
+   will then be updated to store the latest plan scheduled for the day.
+3. The success response is printed to the user through the terminal.
+4. The Day object data is written to the resource file `schedule.txt`.
+
+The following sequence illustrates how the schedule /update command works in greater detail:
+> To simplify the sequence diagram, some method invocations that deemed to be trivial
+> have been removed from the sequence diagram. Reference frames will be elaborated further
+> down this section.
+
+![Update Schedule Sequence Diagram](uml/sequenceDiagrams/images/updateSchedule.png)
```

Comment:
Same as previous comments on `parseUserInput()` activation bars (number 2 and 10 specifically)